redfish-core/include/redfish.hpp

```diff
@@ -242,6 +243,7 @@ class RedfishService
         requestRoutesSubmitTestEvent(app);
 
         requestRoutesHypervisorSystems(app);
+        crow::obmc_dump::requestRoutes(app);
```

Comment:
I guess locks are still in the crow namespace. 
https://github.com/openbmc/bmcweb/blob/6ea9076048028a4adfdbdd2606698bffc81203d3/include/ibm/management_console_rest.hpp#L811

It seems odd, but maybe that is correct. I am good. 

A name other than requestRoutes would be best but again oh well